src/LibVLCSharp/Shared/LibVLCOptions.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LibVLCSharp.Shared
+{
+    /// <summary>
+    /// Fluent builder for constructing LibVLC options.
+    /// Mirrors many common libvlc command line parameters.
+    /// See https://wiki.videolan.org/VLC_command-line_help for more details about available parameters.
+    /// OPTIONS ARE PROVIDED AS-IS, WITHOUT ANY GUARANTEE OF FORWARD/BACKWARD COMPATIBILITY.
+    /// </summary>
+    public class LibVLCOptionsBuilder
```

Comment:
I'd move this class in its own file. I would always have a bad time trying to find that LibVLCOptionsBuilder.cs that doesn't exist...